src/components/involvement/Tabling.jsx

```diff
@@ -6,19 +6,23 @@ import tabling4 from "@/public/images/tabling/tabling_4.webp";
 
 const Tabling = () => {
   return (
-    <div className="flex w-3/4 flex-col items-center gap-y-8">
-      <Subtitle color="text-biscuits-brown-200" text="How to Table" />
+    <div className="flex flex-col items-center gap-y-8">
+      <Subtitle
+        className="w-3/4"
+        color="text-biscuits-brown-200"
+        text="How to Table"
+      />
 
-      <Picture x={20} y={20} image={tabling1} className="px-4 my-4" />
-      <ul className="font-inika text-2xl list-decimal">
+      <Picture x={20} y={20} image={tabling1} className="w-3/4 px-4 my-4" />
```

Comment:
![image](https://github.com/user-attachments/assets/f3b3bddd-3471-4c82-b43b-54b7e8bf04d5)

Width doesn't seem to be doing anything here.
This might be because className in the actual Picture.jsx component takes precedence over this className.

to fix it, add a className prop in picture and insert it to the divider tag.

Reply:
i would also try centering the picture if possible.